tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2365,9 +2363,11 @@ tensorflow::Status ConvertSlice(OpConverterParams* params) {
   const auto& node_def = params->node_def;
   TF_RETURN_IF_ERROR(CheckInputsWeights(
       *params, {{"input", false}, {"begin", true}, {"size", true}}));
-  std::vector<int> begin, size;
-  inputs.at(1).weights().ToVector(&begin);
-  inputs.at(2).weights().ToVector(&size);
+  auto begin_span = inputs.at(1).weights().GetSpan<int>();
+  auto size_span = inputs.at(2).weights().GetSpan<int>();
+  std::vector<int> begin(begin_span.data(),
```

Comment:
Sorry I didn't make it clear: I think we can add a `std::vector<T> ToVector()` method and use `GetSpan` to implement it.

Reply:
This suggestion could be controversial but we can avoid copying the tensor content and let AllLengthsEqual take vector of absl::Span. It should likely work out of the box as {input_dims, begin, size} will be implicitly converted to absl::Span.

Adding ToVector along with documenting that Span should be preferred over Vector to avoid the copy would work as well. Either is okay because this is an internal API and can be refactored easily.

Thoughts?

Reply:
Agree. We should use `GetSpan` whenever possible. But I think we still need a `ToVector` for `size` below since it needs to be modified in line 2395.